drivers/adc/Kconfig.w91

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022-2023 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config ADC_TELINK_W91
+	bool "Telink Semiconductor W91 ADC driver"
+	default y
+	depends on DT_HAS_TELINK_W91_ADC_ENABLED
+	select ADC_CONFIGURABLE_INPUTS
+	help
+	  Enables Telink W91 ADC driver.
+
+if ADC_TELINK_W91
+
+config ADC_TELINK_W91_IPC_RESPONSE_TIMEOUT_MS
```

Comment:
Since we have on N22 side just one listener thread. All IPC drivers timeouts should be the same and equal to max driver timeout.

Reply:
Timeout was replaced

Reply:
New n22 SW revision is added